test/run_tests.bash

```diff
@@ -11,6 +11,12 @@
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 TEST_RESULTS_DIR="${SCRIPT_DIR}/results"
 
+# Ensure we run everything for coverage purposes, but ensure failures are returned by the script
+SUCCESS=0
```

Comment:
We could alternatively just `set -e` or something, but did this in case you want ros coverage to run even when the pytests fail.